extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -53,6 +56,21 @@ const MIN_INTEGER = -100000;
  */
 export let panningMediaPositionDef;
 
+/**
+ * Max distances to keep image in viewport.
+ * left: Percentage between [-50; 50]
+ * right: Percentage between [-50; 50]
+ * top: Percentage between [-50; 50]
+ * bottom: Percentage between [-50; 50]
```

Comment:
Here the left and right values will always be opposite to each other (same thing with top and bottom), so technically we could just store one of them (the positive one) for each axis, and use it to clamp the horizontal/vertical axis on both ways by negating the positive value. Then the Def would just be 2 values: horizontal clamp and vertical clamp